CMakeLists.txt

```diff
@@ -68,14 +85,30 @@ if (FINAL_STAGE GREATER_EQUAL 2)
     MODULE_SOURCES ${SELFHOST_SOURCES}
     RUNTIME_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/runtime"
   )
-  target_include_directories(jakt_stage2 PUBLIC $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/runtime>)
+  target_include_directories(jakt_stage2 PUBLIC
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/runtime>
+    $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/runtime>
+  )
   add_executable(Jakt::jakt_stage2 ALIAS jakt_stage2)
+  apply_output_rules(jakt_stage2)
+endif()
+
+# Link runtime into build directory(ies) for relative pathing usage
+#    Note: "If a sequential execution of multiple commands is required, use multiple execute_process() calls with a single COMMAND argument."
+if (CMAKE_CONFIGURATION_TYPES)
+  foreach (build_type IN LISTS CMAKE_CONFIGURATION_TYPES)
```

Comment:
A `POST_BUILD` command might be better here imo, so the things are linked when necessary. The presence of `CMAKE_CONFIGURATION_TYPES` is not the determining factor for a multi config generator.